### PR TITLE
Add demo for python convai sdk

### PR DESCRIPTION
**Pull request by lacop11**

### CODE REVIEW

File: examples/conversational-ai-python/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+   ```bash
+   poetry install
+   # Or if you don't want to use poetry
+   pip install -e .
+   ```
+
+1. Set up environment variables. You must set the agent ID. The API key is required if the agent is not public ("Enable authentication" in the agent settings).
+
+    ```bash
+    export AGENT_ID=agentid
```
Comment (hikmet-demir) [inline examples/conversational-ai-python/README.md:18]: can they read it from .env file instead?

Comment (hikmet-demir) [reply examples/conversational-ai-python/README.md:18]: we can also add a .env.example

**Review by hikmet-demir — APPROVED** [review]

File: examples/conversational-ai-python/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Conversational AI SDK Python example
+
+## How to run the code
+
+1. Clone this repo
+1. `cd elevenlabs-examples/examples/conversational-ai-python`
+1. Install the dependencies
```
Comment (jodik) [inline examples/conversational-ai-python/README.md:7]: .. dependencies with either

I think many folks not knowing poetry will be confused / just copy paste the whole box below.
Alternatively split into two code blocks.

File: examples/conversational-ai-python/README.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+   poetry install
+   # Or if you don't want to use poetry
+   pip install -e .
+   ```
+
+1. Set up environment variables. You must set the agent ID. The API key is required if the agent is not public ("Enable authentication" in the agent settings).
+
+    ```bash
+    export AGENT_ID=agentid
+    export ELEVENLABS_API_KEY=sk_yourkey
```
Comment (jodik) [inline examples/conversational-ai-python/README.md:19]: put comment above 
# Optional - for auth-enabled agents

**Pull request merged**